local-helm/buyingcatalogue/templates/configmap.yaml

```diff
@@ -5,6 +5,7 @@
 {{- $isapiUrl:= ternary .Values.baseIsapiEnabledUrl  (printf "%s:%d" .Values.isapi.disabledUrl (.Values.isapi.service.port | int )) .Values.isapi.enabled -}}
 {{- $isapiIssuerUrl:= ternary $isapiUrl .Values.isapi.feDisabledIssuerUrl $areAllFEComponentsEnabled -}}
 {{- $adminUrl:= ternary .Values.admin.baseUri .Values.admin.disabledUrl .Values.admin.enabled -}}
+{{- $emailUrl:= printf "%s.%s" (include "call-nested" (list . "email" "email.fullname")) .Release.Namespace -}}
```

Comment:
We could do with a disabled option, allowing us to route to a different service outside the cluster when disabled (as we will eventually need to do when we go to prod)

Reply:
I'll add the option for disabled and keep it blank for now.